packages/sanity/src/structure/panes/document/documentPanel/header/NonReleaseVersionsSelect.tsx

```diff
@@ -91,49 +106,47 @@ export function NonReleaseVersionsSelect(props: {
         </Tooltip>
       ) : null}
 
-      {nonReleaseDropdownOpen && (
```

Comment:
this change wasn't strictly required, but the conditional render here was unnecessary